### ISSUE TITLE
Dependency of cordova-plugin-compat

### ISSUE THREAD

**Issue by CWBudde** [bitpay/cordova-plugin-qrscanner#131]
With the latest update the cordova-plugin-compat has been deprecated. It got integrated into cordova-android@6.3.0. For this reason I would like to suggest to remove the dependency for the QR scanner as well.

**Comment by cedricsteiner** [comment]
Yes, please remove it. It prevents me from packaging on Ionic Pro at the moment. Thanks.

**Comment by Fever905** [comment]
Did something change recently with Cordova or Phone Gap? I had a QR scanner in  my app that now just doesn't load. Scrambling to debug or revert to a prior build.

**Comment by Fever905** [comment]
I think it was a syntax problem on my end. I've had it working for months now.  Thanks tho!

**Comment by bitjson** [comment]
🎉
